nixos/modules/services/misc/weechat.nix

```diff
@@ -0,0 +1,103 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.weechat;
+in
+
+{
+  options.services.weechat = {
+    enable = mkEnableOption "weechat";
+    init = mkOption {
+      description = "Weechat commands applied at start, one command per line.";
+      example = ''
+        /set relay.network.password correct-horse-battery-staple
+        /relay add weechat 9001
+      '';
+      type = types.str;
+      default = "";
+    };
+    root = mkOption {
+      description = "Weechat state directory.";
+      type = types.str;
+      default = "/var/lib/weechat";
+    };
+    scripts = mkOption {
+      description = "Custom scripts to be loaded by the weechat module.";
+      type = with types; listOf (submodule {
+        options = {
+          name = mkOption {
+            type = str;
+            description = "Path to the script in the derivation.";
+          };
+          script = mkOption {
+            type = types.package;
+            description = "Package which contains the actual script.";
+          };
+        };
+      });
+      default = [];
+      example = literalExample ''
+        [
+          { name = "share/jabber.py"; script = pkgs.weechat-xmpp }
+        ]
+      '';
+    };
+    sessionName = mkOption {
+      description = "Name of the `screen' session for weechat.";
```

Comment:
I think it is better to use normal quotes here. I don't think the tex-like backtick/normal tick combination is very common in plain text.

Reply:
not sure though. I've seen this fairly often in the Nix(OS) context including descriptions for options and wanted to be convenient. But feel free to correct me, I'm not sure about this :)

Reply:
You're right, I can find a couple of examples of both simple quotes and `' quotes in `man configuration.nix`. I personally prefer simple quotes but since there is apparently no consensus I'm fine with whatever you prefer.